messages/jsonschema/scripts/templates/java.java.erb

```diff
@@ -0,0 +1,96 @@
+<% @schemas.each do |key, schema| %>
+    public static class <%= class_name(key) %> {
+      <%- schema['properties'].each do |property_name, property| -%>
+        private <%= type_for(class_name(key), property_name, property) -%> <%= property_name %><%= property['items'] ? ' = new java.util.ArrayList<>()' : '' %>;
+      <%- end -%>
+
+        public <%= class_name(key) %>() {}
+
+        public <%= class_name(key) %>(
+        <%- schema['properties'].each_with_index do |(property_name, property), index| -%>
+            <%= type_for(class_name(key), property_name, property) -%> <%= property_name %><%= index < schema['properties'].length-1 ? ',' : ''%>
+        <%- end -%>
+        ) {
+          <%- schema['properties'].each_with_index do |(property_name, property), index| -%>
+            <%- if (schema['required'] || []).index(property_name) -%>
+            this.<%= property_name %> = java.util.Objects.requireNonNull(<%= property_name %>, "<%= class_name(key) %>.<%= property_name %> cannot be null");
+            <%- else -%>
+            this.<%= property_name %> = <%= property_name %>;
+            <%- end -%>
+          <%- end -%>
+        }
+
+      <%- schema['properties'].each do |property_name, property| -%>
+        <%- if (schema['required'] || []).index(property_name) -%>
+        public <%= type_for(class_name(key), property_name, property) -%> get<%= capitalize(property_name) %>() {
+            return java.util.Objects.requireNonNull(<%= property_name %>, "<%= class_name(key) %>.<%= property_name %> cannot be null");
```

Comment:
Getters typically don't have a null check. Ideally null checks are handled in the constructor and setters.

Reply:
Fixed